packages/lib/slots.ts

```diff
@@ -193,7 +193,7 @@ function buildSlotsWithDateRanges({
       : startTimeWithMinNotice;
 
     slotStartTime =
-      slotStartTime.minute() % interval !== 0
+      interval !== 60 && slotStartTime.minute() % interval !== 0
```

Comment:
Now if the user has an schedule starting at 10 but a 5 minute busy slot at 10:00-10:05 the slots would look like that: 
<img width="324" alt="Screenshot 2024-08-22 at 8 55 13 AM" src="https://github.com/user-attachments/assets/d17a5b17-c6d3-4e75-8eac-b8e692d04a88">

Reply:
or other option may be we can decide to say round off (in case) 10:55 to 11:00 or 10:45 to 11:00, but rounding off (ceil) 10:05 to 11:00, i guess we are hiding 55mins of availability. This is just my opinion. 

in case of 10mins ,20mins ,30mins (event length) slots, the ceil may consume max 5-25mins, in worst case 25mins (in case of 30mins eventtypes, host is available from 10:05, but next slot is shown as 10:30). This is as is or currently not affecting 30min slots. This change is only for 60min slots.

May be we can introduce one more setting in eventType to allow user to set max allowable rounding off (or ceil) to next slot or not do any round off. May be we can have new issue for this.

Reply:
I also raised the topic internally with the team. I think we need to be careful with changing the current behavior as this is now what users expect to happen and I also think that many want their hourly events only start at full hours. 

I think an additional setting might be the way to go. Maybe

Reply:
> it will be 10:05 , 12:05, 13:05...

I tested your PR locally and the slots were as shown in the screenshot

Reply:
I also raised the topic internally with the team. I believe we need to be cautious about changing the current behavior, as this is now what users expect. Also, many users probably prefer their hourly events to start at full hours.

An additional setting might be the best approach. Perhaps we could introduce a setting that allows users to choose when the first slot should start, with options for 00, 05, 10, 15, etc., minute slots (defaulting to the current behavior)

Reply:
I recreated the same scenario.
Also there are tests that verify this scenario.

https://www.loom.com/share/a1cd95a7304e47f8bb2234091c8c47c7?sid=0a3c5f37-fd91-4836-8d78-e358c2cc5025

Reply:
Looks like it works differently with third-party calendar events 🤔 my busy time is coming from google calendar